gubernator/view_pr.py

```diff
@@ -183,19 +183,33 @@ def serial(obj):
                     keys = ['repo', 'number'] + list(obj._values)
                     return {k: getattr(obj, k) for k in keys}
                 raise TypeError
-            self.response.write(json.dumps(prs, sort_keys=True, default=serial))
+            self.response.write(json.dumps(prs, sort_keys=True, default=serial, indent=True))
         elif fmt == 'html':
             if user:
                 user = InsensitiveString(user)
                 def acked(p):
-                    if 'lgtm' in p.payload.get('labels', {}):
-                        return True  # LGTM is an implicit Ack
+                    labels = p.payload.get('labels', {})
+                    if 'lgtm' in labels:
+                        if not (user in p.payload.get('approvers', [])
```

Comment:
Why both `not (item in collection)` and `item not in collection` in one `if`?

Reply:
Should this logic be DRYed out and named in a function of `labels` and `approvers`?

Reply:
Fixed. It's not used more than once, so it stays as one line for now.

Reply:
Same logic that you added to `gubernator/filters.py` in this PR

Reply:
Done.